sources/platform/actors/running/actor_standby.md

```diff
@@ -31,6 +31,26 @@ If you're using an Actor built by someone else, see its Information tab to find
 
 Generally speaking, Actors in Standby mode behave as standard HTTP servers. You can use any of the existing [HTTP request methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) like GET, POST, PUT, DELETE, etc. You can pass the input via [HTTP request query string](https://en.wikipedia.org/wiki/Query_string) or via [HTTP request body](https://developer.mozilla.org/en-US/docs/Web/HTTP/Messages#body).
 
+## How do I authenticate my requests
+
+To authenticate requests to Actor Standby, follow the same process as [authenticating requests to the Apify API](../../integrations/programming/api.md).
+You can provide your [API token](../../integrations/programming/api.md#api-token) in one of two ways:
+
+1. _Recommended_: Include the token in the `Authorization` header of your request as `Bearer <token>`. This approach is recommended because it prevents your token from being logged in server logs.
+
+    ```shell
+    curl -H "Authorization: Bearer my_apify_token" \
+      https://rag-web-browser.apify.actor/search?query=apify
+    ```
+
+2. Alternatively, you can append the token as a query parameter named `token` to the request URL.
```

Comment:
Something I realized after writing the previous comment, that we can shorten the second step and omit the `alternatively...` since we are already in numbered list and provide the reader with two choices. Otherwise LGTM

```suggestion
2. Append the token as a query parameter named `token` to the request URL.
```